impress-org/givewp
# Feature: add Per Form Gateways add-on to the form migration process

Resolves [GIVE-60] [GIVE-184]

Related PR: https://github.com/impress-org/give-per-form-gateways/pull/11

## Description



This PR adds the necessary logic to transfer data from the Per Form Gateways addon settings to the migrated forms.

This PR also sets the Per Form Gateways addon as supported by the visual form builder.

## Affects



The migrated forms.

## Visuals



![image](https://github.com/impress-org/givewp/assets/16308864/0059d156-9d89-446b-999a-375a967c0c59)

## Testing Instructions



1. Install the Per Form Gatways addon from this PR/branch: https://github.com/impress-org/give-per-form-gateways/pull/11
2. Create a legacy form and select a few gateways to be displayed when the form gets rendered
3. Do the migration process, then check the Payment Gateways block settings in the migrated form, you should see in the block settings the gateways selected in the previous step, except the ones not compatible with v3 forms.

## Pre-review Checklist






[GIVE-60]: https://stellarwp.atlassian.net/browse/GIVE-60?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[GIVE-184]: https://stellarwp.atlassian.net/browse/GIVE-184?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
